### PR TITLE
code review practice

### PR DESCRIPTION
**Pull request by Abass-rashid**
in this PR I have copied one of my previous microverse projects to practice code review on it

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        <li>
+          HTML and css
+        </li>
+        <li>
+          javascript
+        </li>
+        <li>
+          python
+        </li>
+      </ul>
```
Comment (Abass-rashid) [inline index.html:50]: Kindly use links or buttons for modals

File: style.css [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+@media (max_width:460px) {
+  body {
+    background-color: gray;
+    color: white;
+  }
+}
+
+#nav-bar {
```
Comment (Abass-rashid) [inline style.css:15]: I think it would be better to use a class selector instead of id for CSS best practice

File: style.css [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+.nav-link {
+  display: flex;
+  align-items: center;
+  justify-content: space-around;
+}
+
+#header-img {
+  max-width: 70%;
+  display: block;
```
Comment (Abass-rashid) [inline style.css:49]: I think it would be better to use a class selector instead of id for CSS best practice

**Comment by Abass-rashid** [comment]
HI @Abass-rashid 
♻️ ### **Changes** Required

1. Kindly use linters in your project for easy debugging :
2. Kindly document your README file in a descriptive way
3. kindly check inline comment for other corrections

File: index.html [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    </section>
+    <video id="video" width="320" height="240" controls>
+      <source src="movie.mp4" type="video/mp4">
+
+    </video>
+
+
```
Comment (ishimwezachee) [inline index.html:57]: - [ ] You should review these unrequited spaces and explain why its important to remove them to a student.

**Review by ishimwezachee — CHANGES_REQUESTED** [review]
Hi @Abass-rashid >, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights


## Required Changes ♻️

- [ ] You should review your project documentation for it is missing some important part as the description, and explain to a student to make those changes. 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        <li>
+          python
+        </li>
+      </ul>
+    </section>
+    <video id="video" width="320" height="240" controls>
+      <source src="movie.mp4" type="video/mp4">
+
+    </video>
+
```
Comment (Abass-rashid) [inline index.html:56]: Kindly avoid unnecessary  blank space for  HTML best practice

**Review by Abass-rashid — COMMENTED** [review]
HI @Abass-rashid 

1. kindly add linters in your project for easy debugging
2.  Kindly document your README  file in a descriptive way
3. Please  look in the inline-comment for other correction

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Abass-rashid,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**